### DISCUSSION TITLE
Unable to login to Argo Dashboard

### DISCUSSION THREAD

**Discussion by rajendragosavi** [argoproj/argo-workflows#6170] [Q&A]
Hi,

I am facing an issue in logging into the argo UI Dashboard.

Brief about my 

setup.

I EKS Cluster running and I have deployed argo workflow using - [https://github.com/argoproj/argo-workflows/tree/stable/manifests/cluster-install](https://github.com/argoproj/argo-workflows/tree/stable/manifests/cluster-install).

I am exposing the argo-server as loadbalancer service in a private subnet and exposing it using nginx-reverse to the public internet.

I am facing an error when I am trying to login to the dashboard. Following the UI screenshot. 

I am not understanding which argo UI token I need to log in. I have tried with argo-secret, but it still does not work.

<img width="1792" alt="Screenshot 2021-06-17 at 12 19 35 PM" src="https://user-images.githubusercontent.com/36725053/122346224-5aaf8c00-cf66-11eb-8c38-7caff615c334.png">

Plese guide me if I am missing something.

@alexec

**Comment by alexec** [comment]
Try logging out and and back in.
